identifiers/country-pa/README

```diff
@@ -0,0 +1,9 @@
+README
```

Comment:
Lets keep the title consistent with other directories (example [Canada](https://github.com/opencivicdata/ocd-division-ids/blob/master/scripts/country-ca/README.md?plain=1#L1)).
```suggestion
# Open Civic Data Divisions: Panama
```